h2o-r/h2o-package/R/frame.R

```diff
@@ -2864,6 +3163,13 @@ h2o.ischaracter <- function(x) {
 #' @name h2o.asfactor
 #' @param x An H2OFrame object.
 #' @seealso \code{\link[base]{as.factor}} for the base R implementation.
```

Comment:
this "see also" points to itself. Can you change this since you're already working in this file :)
>#' @seealso \code{\link[base]{as.numeric}} for the base R implementation.